### DISCUSSION TITLE
Transferring a domain

### DISCUSSION THREAD

**Discussion by Lancito01** [vercel/community#6402] [Help] [answered]
### Summary

So I was looking into hosting a React Vite website for myself. I wanted to get the domain lancito01.dev, and I looked into Vercel for its ease-of-use and deployment, since it's linked to my GitHub, where the files are uploaded. Then, I looked into another domain seller (NameCheap), and I saw the domain was cheaper, so me not knowing any better, I bought it thinking I could host it through Vercel. Much to my surprise, this was not possible and I needed to transfer the website to Vercel and pay the original 20$ that I needed to pay. I started the transfer from NameCheap to Vercel, and I was wondering how long it'd take to see my website up and running on the domain I got?

### Steps to Reproduce

.

**Comment by versecafe** [comment, marked as answer]
@Lancito01 Changing name servers usually takes a few hours to kick in globally, but DNS record changed can be even faster, but either way you should be able to have your site up in under a day on your new domain on Vercel.

**Reply by Lancito01** [reply]
Gotcha. I Initiated the transfer around ~4-5 hours ago, so I guess I'll check again when I wake up tomorrow. In case something goes wrong, where can I ask about it? Here?

**Reply by versecafe** [reply]
Asking on here is a good place to start, if you're still having issues just ping me and I can help guide you through to make sure everything is configured correctly or you can reach out to the Vercel team.
